settings.py

```diff
@@ -593,6 +593,9 @@
 
 # Setting this to False will make the system process outgoing and incoming SMS
 # immediately rather than use the queue.
+# This should always be set to True in production environments, and the sms_queue
+# celery worker(s) should be deployed. We set this to False for tests and (optionally)
+# for local testing.
 SMS_QUEUE_ENABLED = False
```

Comment:
Should this be set to `True` here and `False` in [testsettings.py](https://github.com/dimagi/commcare-hq/blob/1d8bea20957f0cc97280a5857d095a558b8fa8b0/testsettings.py) and [dev_settings.py](https://github.com/dimagi/commcare-hq/blob/1d8bea20957f0cc97280a5857d095a558b8fa8b0/dev_settings.py) with a comment in dev_settings about why you might want to change it to `True` in your localsettings.py? That way production environments would automatically default to `True`.

Reply:
agree, updated